modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/AbstractPageMemoryIndexStorage.java

```diff
@@ -368,7 +368,7 @@ private boolean advanceIfNeededBusy() throws IgniteInternalCheckedException {
         }
     }
 
-    protected <V> V busy(Supplier<V> supplier) {
+    protected <T> T busy(Supplier<T> supplier) {
```

Comment:
Meow

Reply:
There is `V` on the class itself, so this method-scoped `V` shadows that class-scoped `V`, Idea highlights this with yellow. It's probably a good thing to not have such shadowings to prevent confusion.